ui/pages/swaps/swaps.util.js

```diff
@@ -93,6 +96,9 @@ const getBaseApi = function (
       return `${baseUrl}/aggregatorMetadata`;
     case 'gasPrices':
       return `${baseUrl}/gasPrices`;
+    case 'suggestedGasFees':
+      // We will probably use the hook "useGasFeeEstimates" instead of this.
+      return `${GAS_API_BASE_URL}/networks/${chainIdDecimal}/suggestedGasFees`;
```

Comment:
I can probably remove this since we use `useGasFeeEstimates` in Swaps, which calls this endpoint.